landscape.yml

```diff
@@ -486,67 +486,42 @@ landscape:
             description: |
               Open source SDN Controller with largest ecosystem and deployment footprint.
           - item:
-            name: Tungsten Fabric
-            homepage_url: https://tungsten.io/
-            logo: tungstenfabric-stacked-color.svg
+            name: Nephio
+            homepage_url: https://nephio.org
+            logo: Nephio_logo_colorblack_horizontal.svg
+            crunchbase: https://www.crunchbase.com/organization/nephio
             twitter: https://twitter.com/LF_Networking
-            crunchbase: https://www.crunchbase.com/organization/tungsten-fabric
-            repo_url: https://github.com/tungstenfabric/ci-management
+            repo_url: https://github.com/nephio-project
```

Comment:
This should be the url of a repository, not the project. I'd recommend changing this to `https://github.com/nephio-project/nephio` and removing `/nephio-project/nephio` from the `additional_repos` list.

Reply:
Thanks for catching the duplicates and the explanation of the URL.  I've resolved the things you've identified.